.vimrc.bundles.default

```diff
@@ -1,14 +1,62 @@
+" Config files {
+    " All config variables handle invisibility for you.
```

Comment:
Should I split this part into another file? As it stands it is duplicated in .vimrc because .vimrc is not sourced at bootstrap.